examples/README.md

```diff
@@ -0,0 +1,113 @@
+# Examples
+
+this directory contains examples for a few features of the autograder
+
+## Assessments
+
+The Hello folder and Hello.tar contain various examples of different styles of Assessments
+
+
+## Importing Assessments
+
+whether you are importing one or more assessments the procedure is the same
+
+multiple assessment import starter.tar is the starter file for importing one or more assessments
+
+
+```bash
+multiple assessment import starter.tar
+├── assessment1
+│   ├── assessment1.yml
+│   ├── autograde.tar
+│   └── autograde-Makefile
+└── assessment2
+    └── assessment1.yml
+
+```
+
+every folder in the tar represents a separate assessment
+
+assessment1 is an example of an assessment with an autograding component
+
+properties.yml
+```YAML
+---
+general:
+  name: assessment1
+  description: ''
+  display_name: ASSESSMENT WITH AUTOGRADING
+  handin_filename: handin.zip
+  handin_directory: handin
+  max_grace_days: 0
+  handout: ''
+  writeup: ''
+  max_submissions: -1
+  disable_handins: false
+  max_size: 2
+  has_svn: false
+  category_name: Lab Activities
+handin:
+  start_at: '1523557920'
+  visible_at: '1523557967'
+  due_at: '1543692720'
+  end_at: '1543699920'
+  grading_deadline: '1543693720'
```

Comment:
Hmmm.  I know I said this is a programmatic interface to me, but ... this isn't very readable, I don't think.

What do you think of ISO 8601 timestamps of the form YYYY-mm-ddTHH:MM:SS instead?  (E.g., 2021-02-01T00:00:00).

Reply:
the problem i was having was with timezones and DST i know a bunch of languages handle time differently and ruby isnt very good at playing with others i chose this to put the least amount of strain on the ruby time parser

Reply:
`cal 9 1752` says it all

Reply:
it seems to work fine so its this format now